Doppler.HelloMicroservice/appsettings.json

```diff
@@ -1,10 +1,16 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Information",
+      "Default": "Debug",
       "Microsoft": "Warning",
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "LogglyConfig": {
+    "CustomerToken": "REPLACE_WITH_CUSTOMER_TOKEN"
+  },
+  "Serilog": {
+    "MinimumLevel": "Warning"
```

Comment:
Por que este es diferente que el LogLevel por defecto?

Reply:
Here it should be information